src/sampling/homogeneous.jl

```diff
@@ -40,6 +40,24 @@ function sample(rng::AbstractRNG, triangle::Triangle{Dim,T}, method::Homogeneous
   (randpoint() for _ in 1:(method.size))
 end
 
+function sample(rng::AbstractRNG, ball::Ball{Dim,T}, method::HomogeneousSampling) where {Dim,T}
+  if Dim == 2
+    function randpoint2()
+      u₁, u₂ = rand(rng, T, 2)
+      ball(sqrt(u₁), u₂)
```

Comment:
Hi @ErickChacon can you clarify the sqrt?

Reply:
Yes. In order to simulate homogeneusly, we need the marginal distributions for the radio and the angle. It can be demonstrated that the marginal distribution for the radio is $2r/R^2$ which can be simulated as $\sqrt{U}$ using the inverse of the cumulative distribution function.